canonical-web-and-design/ubuntu.com
# Update 3rd party geolocation PDF link

## Done

Changed the PDF on Geolocation.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare it against the [copy doc](https://docs.google.com/document/d/1AlOE2_-BqZNP4hiiuf8kZney8g1zvqAlDYl2u3X6m6I/edit).


## Issue / Card

Fixes #4234

## Screenshots

[If relevant, please include a screenshot.]
